src/priority.c

```diff
@@ -94,18 +84,24 @@ stumpless_prival_from_string( const char *string ) {
 
   // Calculate the severity length
   len = slen - ++len;
-  pre_len = sizeof(pre_severity) - 1;
+//  pre_len = sizeof(pre_severity) - 1;
 
-  param = alloc_mem( len + 1 + pre_len );
+  // Copy the severity substring to the param buffer
+  param = copy_cstring_length( ++period, len );
   if( !param ) {
     return -1;
   }
+
+//  param = alloc_mem( len + 1 + pre_len );
```

Comment:
remove these blocks of commented code as well

Reply:
done